doc/classes/AudioStreamPlayer2D.xml

```diff
@@ -139,30 +144,40 @@
 			<return type="void">
 			</return>
 			<description>
+				Stops the audio.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="area_mask" type="int" setter="set_area_mask" getter="get_area_mask">
+			Areas in which this sound plays.
 		</member>
 		<member name="attenuation" type="float" setter="set_attenuation" getter="get_attenuation">
+			Dampens audio over distance with this as an exponent.
 		</member>
 		<member name="autoplay" type="bool" setter="set_autoplay" getter="is_autoplay_enabled">
+			If [code]'true'[/code], audio plays from start of the game. Default value: [code]'false'[/code].
 		</member>
 		<member name="bus" type="String" setter="set_bus" getter="get_bus">
+			Bus on which this audio is playing.
 		</member>
 		<member name="max_distance" type="float" setter="set_max_distance" getter="get_max_distance">
+			Sets the absolute maximum of the soundlevel, in dB.
```

Comment:
I'm not sure if the description is accurate.

Reply:
oh, weird, whatever i put there doesn't make much sense